src/EditorFeatures/VisualBasicTest/Structure/MetadataAsSource/RegionDirectiveStructureTests.vb

```diff
@@ -28,7 +28,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Outlining.Metadata
 #End Region|}
 "
             Await VerifyBlockSpansAsync(code,
-                Region("span", "Assembly mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", autoCollapse:=True, isDefaultCollapsed:=False))
+                Region("span", "Assembly mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", autoCollapse:=True, isDefaultCollapsed:=True))
```

Comment:
🤔 The Metadata As Source view was designed to keep this comment visible, but this change will cause it to collapse.

Reply:
It appears that ship has already sailed. This is from 17.1.5 with "enter outlining mode" turned on:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/754264/166402731-56dd1da3-e18d-4970-a276-9da9c75b9466.png">

I would argue that if that was the intended behaviour, using a region seems odd, and it should have just been three lines of comments, not two. In light of the fact that we now show embedded and sourcelink source with the same metadata views, special casing regions in those views makes even less sense.